roofit/roofitcore/test/CMakeLists.txt

```
@@ -53,6 +53,8 @@ if(NOT MSVC OR win_broken_tests)
   ROOT_ADD_GTEST(testRooPolyFunc testRooPolyFunc.cxx LIBRARIES Gpad RooFit)
 endif()
 if (roofit_multiprocess)
+  ROOT_ADD_GTEST(TestStatisticsPlot TestStatistics/testPlot.cpp LIBRARIES RooFitMultiProcess RooFitCore RooFit 
+                 COPY_TO_BUILDDIR ${CMAKE_CURRENT_SOURCE_DIR}/TestStatistics/TestStatistics_ref.root)
```

```suggestion
  ROOT_ADD_GTEST(testTestStatisticsPlot TestStatistics/testPlot.cpp LIBRARIES RooFitMultiProcess RooFitCore RooFit 
                 COPY_TO_BUILDDIR ${CMAKE_CURRENT_SOURCE_DIR}/TestStatistics/TestStatistics_ref.root)
```
The test binaries are prefixed by `test` by convention (I know `testTest` looks weird, but we are testing test statistics so that's how it is).